exercises/concept/language-list/.docs/instructions.md

```diff
@@ -60,11 +60,11 @@ LanguageList.new()
 
 ## 6. Define a function to determine if the list is exciting
```

Comment:
Can you also update the this headline and its twin brother in `hints.md`?

Reply:
done